src/CommunityToolkit.Maui.Core/Handlers/Popup/PopUpViewHandler.android.cs

```diff
@@ -0,0 +1,126 @@
+	/// <param name="handler">An instance of <see cref="PopupViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">We don't need to provide the result parameter here.</param>
+	public static void MapOnOpened(PopupViewHandler handler, IPopup view, object? result)
+	{
+		handler.NativeView?.Show();
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup is LightDismissed.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">The result that should return from this Popup.</param>
+	public static void MapOnLightDismiss(PopupViewHandler handler, IPopup view, object? result)
+	{
+		if (view.IsLightDismissEnabled)
+		{
+			view.LightDismiss();
+		}
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.Anchor"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapAnchor(PopupViewHandler handler, IPopup view)
+	{
+		handler.NativeView?.SetAnchor(view);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.IsLightDismissEnabled"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapLightDismiss(PopupViewHandler handler, IPopup view)
+	{
+		handler.NativeView?.SetLightDismiss(view);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.Color"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapColor(PopupViewHandler handler, IPopup view)
+	{
+		handler.NativeView?.SetColor(view);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.Size"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapSize(PopupViewHandler handler, IPopup view)
+	{
+		ArgumentNullException.ThrowIfNull(handler.Container);
+
+		handler.NativeView?.SetSize(view, handler.Container);
+		handler.NativeView?.SetAnchor(view);
+	}
+
+	/// <inheritdoc/>
+	protected override MauiPopup CreateNativeElement()
+	{
+		_ = MauiContext ?? throw new InvalidOperationException("MauiContext is null, please check your MauiApplication.");
+		_ = MauiContext.Context ?? throw new InvalidOperationException("Android Context is null, please check your MauiApplication.");
+
+		return new MauiPopup(MauiContext.Context, MauiContext);
```

Comment:
You can use Context directly. Instead if MauiContext.Context

Reply:
We don't have the `Context` here, because the Popup extends from `ElementHandler` and not from `ViewHandler`